doc/user/content/sql/create-sink.md

```diff
@@ -62,12 +62,27 @@ Field | Value | Description
 `ssl_key_password` | `text` | Your SSL key's password, if any.
 `ssl_ca_location` | `text` | The absolute path to the certificate authority (CA) certificate. Used for both SSL client and server authentication. If unspecified, uses the system's default CA certificates.
 
-### AS OF
+#### Kerberos `WITH` options
+
+Use the following options to connect Materialize to an Kerberized Kafka
+cluster.
+
+Field | Value | Description
+------|-------|------------
+`sasl_kerberos_keytab` | `text` | The absolute path to your keytab.
```

Comment:
Do we have a stylistic convention not to use personal pronouns? "The absolute path to the keytab" works better, if so.

Reply:
We actually have the opposite convention, which is to use personal pronouns liberally.